### PR TITLE
Sd/custom template

### PR DESCRIPTION
**Pull request by stduong**

### CODE REVIEW

File: templates/default/nova.conf.erb [hunk 1]
```diff
@@ -5,6 +5,10 @@
     <% if value.class == Hash -%>
 <%= "# #{value['comment']}" -%>
 <%= key %> = <%= value['set_to'] %>
+    <% elsif value.class == Array -%>
+    <% for list_value in value %>
```
Comment (mikeurbach) [inline templates/default/nova.conf.erb]:
```
<% value.each do |item| %>
```
syntax is a little more idomatic?

**Review by mikeurbach — APPROVED** [review]
minor suggestion

File: templates/default/nova.conf.erb (hunk 1 above)
Comment (automatedops) [reply templates/default/nova.conf.erb]: Sorry. My python guy.

**Review by automatedops — APPROVED** [review]
Steven will update it.

File: templates/default/nova.conf.erb [hunk 2]
```diff
@@ -5,6 +5,10 @@
     <% if value.class == Hash -%>
 <%= "# #{value['comment']}" -%>
 <%= key %> = <%= value['set_to'] %>
+    <% elsif value.class == Array -%>
+    <% value.each do |list_value| %>
```
Comment (girak) [inline templates/default/nova.conf.erb]: indent?

**Review by girak — APPROVED** [review]

**Pull request merged**